tools/install.py

```diff
@@ -122,7 +122,8 @@ def corepack_files(action):
   })
 
   # On z/OS, we install node-gyp for convenience as native add-ons
-  # are dependend on
+  # are dependend on, as some vendors don't have external access
+  # and may want to build their own native node.js modules
```

Comment:
```suggestion
  # and may want to build native addon modules.
```
because the reason applies to any native addon, not just their own.